pkg/operator/configobserver/featuregates/observe_featuregates_test.go

```diff
@@ -47,23 +47,20 @@ func TestObserveFeatureFlags(t *testing.T) {
 			name:        "default",
 			configValue: configv1.Default,
 			expectedResult: []string{
-				"ExperimentalCriticalPodAnnotation=true",
```

Comment:
has been those features removed?

I checked https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ but it's not clear for me.

Reply:
@sttts is this okay?

Reply:
I found out from this file list of enabled/disabled feature gates

https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L101-L110